### ISSUE TITLE
If this project is live ?

### ISSUE THREAD

**Issue by maver1ck** [databricks/click#69]
Dear Databricks guys,
You opensourced great project but after that there is no actions taken. 
PR are waiting. Issues are not resolved.
So the question is: are you planning to maintain this project?

Regards,
Maciek

**Comment by maver1ck** [comment]
CC: @nicklan

**Comment by nicklan** [comment]
Yes, it's alive!  I'm sorry for the lack of response recently.  I will be getting back to responding to things in the very near future!

**Comment by rawkode** [comment]
@nicklan Perhaps it would be useful to delegate certain permissions for merging and tagging labels to some community members that wish to help.

There's still very little activity on issues and pull requests, even though people are trying to contribute.

**Comment by maver1ck** [comment]
@nicklan 
Two months later and still nothing happens.

**Comment by rawkode** [comment]
@maver1ck I'm happy to contribute time to a fork of this, should you be interested in helping ?

I don't think we're going to get any further response here

**Comment by milesgranger** [comment]
I was also going to contribute to a small issue I was having, but seeing PRs being left without activity doesn't make me confident the time will be well spent.

**Comment by nicklan** [comment]
Hey all,

I've been a little overloaded with work and life and therefore remiss in maintaining Click, and for that I apologize.  I've explicitly added time to my schedule to be a better maintainer, and am working through the backlog of things.

@rawkode I'd love to delegate and get more maintainers involved.  Let me have a look at who's contributed and I'll figure out what we can sort out on that front.

**Comment by rawkode** [comment]
@nicklan welcome back 😃

**Comment by nicklan** [comment]
@asychev yeah, and I've been going over PRs and merging them.  Was there something specific you wanted addressed?  Sorry if I haven't gotten to something important

**Comment by maver1ck** [comment]
@nicklan One year later and no support at all.
Maybe it's time to give access to this project to community ?

**Comment by asychev** [comment]
@maver1ck Unfortunately, this project is really abandoned. Not working with k8s 1.17 at all (Hyper error: invalid domain character). It's a pity...
